### DISCUSSION TITLE
6.x 配置文件里加了日志 并不生效

### DISCUSSION THREAD

**Discussion by nbnatcom** [w7corp/easywechat#2813] [General]
` /**
     * 日志配置
     *
     * level: 日志级别, 可选为：
     *         debug/info/notice/warning/error/critical/alert/emergency
     * path：日志文件位置(绝对路径!!!)，要求可写权限
     */

    'logging' => [
        'default' => 'dev', // 默认使用的 channel，生产环境可以改为下面的 prod
        'channels' => [
            // 测试环境
            'dev' => [
                'driver' => 'single',
                'path' => __DIR__.'\log\easywechat.log',
                'level' => 'debug',
            ],
            // 生产环境
            'prod' => [
                'driver' => 'daily',
                'path' => __DIR__.'\log\easywechat.log',
                'level' => 'info',
            ],
        ],
    ],`

这样的配置,不管怎么操作 并没有生成任何日志文件
帮助文件easywechat\docs\src\6.x\sidebar.js 都将通用里的日志注释了 是用不了了吗

**Comment by oydm** [comment]
同样遇到的问题，先mark

**Comment by oydm** [comment]
同样遇到的问题，先mark
